extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/DevServicesKafkaProcessor.java

```diff
@@ -253,6 +288,7 @@ private RedPandaKafkaContainer(DockerImageName dockerImageName, int fixedExposed
             this.port = fixedExposedPort;
             withNetwork(Network.SHARED);
             withExposedPorts(KAFKA_PORT);
+            withLabel(DEV_SERVICE_LABEL, DEV_SERVICE_NAME);
```

Comment:
Should be added only in dev mode, and not in test mode?

Reply:
Fixed.